src/main/java/fr/silverpricing/api/model/CategoryChambre.java

```diff
@@ -1,6 +1,7 @@
 package fr.silverpricing.api.model;
```

Comment:
:information_source: Package name 'service' is too generic and should be more specific. (Severity: Minor)